### ISSUE TITLE
funds on hold forever

### ISSUE THREAD

**Issue by vitorego** [DeviaVir/zenbot#244]
Today i got this error until a shutdown the bot and restart it again. 

```
2017-06-08 16:01:25 - funds on hold after cancel, waiting 5s
2017-06-08 16:01:23  2424.74 BTC-EUR    0.0%        3    --       -0.1881   0.0500  selling  0.29642 BTC  297.46 EUR    +0.6%   +5.0%
2017-06-08 16:01:31 - funds on hold after cancel, waiting 5s

2017-06-08 16:01:37 - funds on hold after cancel, waiting 5s

2017-06-08 16:01:43 - funds on hold after cancel, waiting 5s

2017-06-08 16:01:48 - funds on hold after cancel, waiting 5s
2017-06-08 16:01:50  2424.71 BTC-EUR    0.0%        5    --       -0.1882   0.0500  selling  0.29642 BTC  297.46 EUR    +0.6%   +5.0%
2017-06-08 16:01:55 - funds on hold after cancel, waiting 5s
2017-06-08 16:01:50  2424.71 BTC-EUR    0.0%        5    --       -0.1882   0.0500  selling  0.29642 BTC  297.46 EUR    +0.6%   +5.0%
2017-06-08 16:02:00 - funds on hold after cancel, waiting 5s

2017-06-08 16:02:06 - funds on hold after cancel, waiting 5s

2017-06-08 16:02:12 - funds on hold after cancel, waiting 5s

2017-06-08 16:02:17 - funds on hold after cancel, waiting 5s

2017-06-08 16:02:23 - funds on hold after cancel, waiting 5s
2017-06-08 16:02:12  2424.99 BTC-EUR    0.0%        5    --       -0.1866   0.0500  selling  0.29642 BTC  297.46 EUR    +0.6%   +5.0%
2017-06-08 16:02:29 - funds on hold after cancel, waiting 5s

2017-06-08 16:02:34 - funds on hold after cancel, waiting 5s

2017-06-08 16:02:41 - funds on hold after cancel, waiting 5s

2017-06-08 16:02:46 - funds on hold after cancel, waiting 5s

2017-06-08 16:02:52 - funds on hold after cancel, waiting 5s
2017-06-08 16:02:36  2433.26 BTC-EUR   +0.3%        6    --       -0.1385   0.0500  selling  0.29642 BTC  297.46 EUR    +0.8%   +4.9%
2017-06-08 16:02:58 - funds on hold after cancel, waiting 5s

2017-06-08 16:03:04 - funds on hold after cancel, waiting 5s

2017-06-08 16:03:10 - funds on hold after cancel, waiting 5s

2017-06-08 16:03:15 - funds on hold after cancel, waiting 5s

2017-06-08 16:03:22 - funds on hold after cancel, waiting 5s
2017-06-08 16:03:15  2430.17 BTC-EUR   +0.2%        7    --       -0.1565   0.0500  selling  0.29642 BTC  297.46 EUR    +0.7%   +4.9%
2017-06-08 16:03:28 - funds on hold after cancel, waiting 5s

```

**Comment by carlos8f** [comment]
I "think" I fixed this with 17506db8ad172da5faaf25ece6214ab6c2ebad6b , it would happen after a partial fill. Another reason this might happen is if you use `--sell_pct=100`, since GDAX holds a little extra than the actual order total. `--sell_pct=99` would fix it.

**Comment by alincalinciuc** [comment]
this is still happening
